lib/ui/account/password_reentry_page.dart

```diff
@@ -169,6 +170,7 @@ class _PasswordReentryPageState extends State<PasswordReentryPage> {
                 Padding(
                   padding: const EdgeInsets.fromLTRB(20, 24, 20, 0),
                   child: TextFormField(
+                    key: const ValueKey("passwordInputField"),
```

Comment:
what's the benefit of adding these valueKeys. By default, it seems to be missing in most tutorials/examples.